### DISCUSSION TITLE
[RAWeb API] `rating` and `ratingVoteCount` for games

### DISCUSSION THREAD

**Discussion by TapiocaFox** [RetroAchievements/RAWeb#2623] [Ideas]
It would be nice to have `rating` and `ratingVoteCount` fields in responses that include game information. For example in Daijishou (`⭐ 4.90 (89 votes).`):

![telegram-cloud-photo-size-5-6201853550913503919-y](https://github.com/user-attachments/assets/31e09471-1aa9-4655-82dc-65db1a7d4daa)

**Comment by wescopeland** [comment]
Hi @TapiocaFox! Unfortunately, RetroAchievements has discontinued its game rating feature. It will likely return in some form in the future, but will look very different than the star rating system we previously had.

It may be best to source this data from other game databases.

**Reply by TapiocaFox** [reply]
That is unfortunate, I like this feature very much. But thanks for the answer! Looking forward to the return in new form!
